docs/usage.md

```diff
@@ -69,25 +104,38 @@ results         # Finished results (configurable, see below)
+    * [Extra Documentation on annotation](docs/annotation.md)
+
 ### Updating the pipeline
-When you run the above command, Nextflow automatically pulls the pipeline code from GitHub and stores it as a cached version. When running the pipeline after this, it will always use the cached version if available - even if the pipeline has been updated since. To make sure that you're running the latest version of the pipeline, make sure that you regularly update the cached version of the pipeline:
+
+When you run the above command, Nextflow automatically pulls the pipeline code from GitHub and stores it as a cached version.
+When running the pipeline after this, it will always use the cached version if available - even if the pipeline has been updated since.
+To make sure that you're running the latest version of the pipeline, make sure that you regularly update the cached version of the pipeline:
 
 ```bash
 nextflow pull nf-core/sarek
 ```
 
 ### Reproducibility
-It's a good idea to specify a pipeline version when running the pipeline on your data. This ensures that a specific version of the pipeline code and software are used when you run your pipeline. If you keep using the same tag, you'll be running the same version of the pipeline, even if there have been changes to the code since.
 
-First, go to the [nf-core/sarek releases page](https://github.com/nf-core/sarek/releases) and find the latest version number - numeric only (eg. `1.3.1`). Then specify this when running the pipeline with `-r` (one hyphen) - eg. `-r 1.3.1`.
+It's a good idea to specify a pipeline version when running the pipeline on your data.
+This ensures that a specific version of the pipeline code and software are used when you run your pipeline.
+If you keep using the same tag, you'll be running the same version of the pipeline, even if there have been changes to the code since.
 
-This version number will be logged in reports when you run the pipeline, so that you'll know what you used when you look back in the future.
+First, go to the [nf-core/sarek releases page](https://github.com/nf-core/sarek/releases) and find the latest version number - numeric only (eg. `2.5.0`).
+Then specify this when running the pipeline with `-r` (one hyphen) - eg. `-r 2.5.0`.
 
+This version number will be logged in reports when you run the pipeline, so that you'll know what you used when you look back in the future.
 
 ## Main arguments
 
 ### `-profile`
-Use this parameter to choose a configuration profile. Profiles can give configuration presets for different compute environments. Note that multiple profiles can be loaded, for example: `-profile docker` - the order of arguments is important!
+
+Use this parameter to choose a configuration profile.
+Profiles can give configuration presets for different compute environments.
+Note that multiple profiles can be loaded, for example: `-profile docker` - the order of arguments is important!
```

Comment:
The example doesn't load multiple profiles. And also "the order of arguments is important" leaves a bit too much to the imagination. But I guess there is no easy answer to what difference the order does?

Reply:
Ah, this should be adressed in the template I guess. Feel free to ignore and blame someone else. :)

Reply:
This is totally template stuff. I do blame Orlando for this ;-)